src/player/C4Player.cpp

```diff
@@ -43,8 +43,10 @@
 #include "object/C4ObjectMenu.h"
 #include "platform/C4GamePadCon.h"
 #include "player/C4PlayerList.h"
+#include "player/C4PlayerScript.h"
+#include "script/C4AulDefFunc.h"
 
-C4Player::C4Player() : C4PlayerInfoCore()
+C4Player::C4Player() : C4PlayerInfoCore(), C4PropList(GetPropListPrototype(C4PlayerScript::PROTOTYPE_NAME))
```

Comment:
I'd add something like

```c
const Player = new _Player
{
// ...
};
```

in System.ocg and then inherit from `Player` here instead. This way, we can implement player utility functions such as `GetTaggedPlayerName` transparently in script.

Reply:
Done!